--- a/package/pkg-generic.mk
+++ b/package/pkg-generic.mk
@@ -185,7 +185,7 @@ endif
 # Generic package build steps
 # INNER - True if we are being called from the generic package infrastructure
 # OUTER - True if we are being called from the generic package infrastructure from the outer build make process
-TOOLCHAIN_EXTERNAL_GCC_VERSIONS := 4.3 4.4 4.5 4.6 4.7 4.8 4.9 5 6 7 8 9
+TOOLCHAIN_EXTERNAL_GCC_VERSIONS := 4.3 4.4 4.5 4.6 4.7 4.8 4.9 5 6 7 8 9 10 11 12 13 14 15
 
 # Check external toolchain libc version
 TOOLCHAIN_EXTERNAL_LIBC_VERSIONS := glibc musl uclibc
